src/Mvc/Mvc.RazorPages/src/Infrastructure/PageActionDescriptorProvider.cs

```diff
@@ -132,16 +127,28 @@ private static string TransformPageRoute(PageRouteModel model, SelectorModel sel
                 return selectorModel.AttributeRouteModel.Template;
```

Comment:
Should it work in this case?

Reply:
Probably should, it would enable devs to create templates with tokens in their custom configured routing model for specific page. I will dig into it :)